src/pages/SubmissionHistory/SubmissionHistory.js

```diff
@@ -123,16 +134,26 @@ export const SubmissionHistory = () => {
                 paddingTop: '1em'
               }),
               columns: [
-                ...isActionMenuEnabled() ?
+                ...getConfig().isActionMenuEnabled ?
                   [{
                     size: { basis: 100, grow: 0 },
                     field: 'actions',
-                    headerRenderer: () => h(TextCell, ['Actions']),
-                    cellRenderer: () => {
-                      return div(
-                        { style: { textAlign: 'center' } },
-                        [icon('cardMenuIcon', { size: 24, onClick: () => { window.alert('TODO: go to actions menu') } })]
-                      )
+                    headerRenderer: () => h(TextCell, {}, ['Actions']),
+                    cellRenderer: ({ rowIndex }) => {
+                      return h(MenuTrigger, {
+                        'aria-label': 'Action selection menu',
+                        content: h(Fragment, [
+                          h(MenuButton, {
+                            style: { fontSize: 15 },
+                            onClick: () => { cancelRunSet(paginatedPreviousRunSets[rowIndex].run_set_id) }
+                          }, ['Abort'])
+                        ])
+                      }, [
+                        h(Clickable, {
+                          style: { textAlign: 'center' },
+                          'aria-label': 'Action selection menu'
+                        }, [icon('cardMenuIcon', { size: 35 })])
```

Comment:
I am wondering if it would be good to reduce the spacing between the menu and the actions button? Based on the screenshot it feels that they are a bit detached. What do you think?

Reply:
I think that will look better visually.